tests/conftest.py

```diff
@@ -1,24 +1,78 @@
 import os
 from typing import List, Optional, Tuple
 
+from PIL import Image
 import pytest
 import torch
-from transformers import AutoModelForCausalLM
+from transformers import AutoModelForCausalLM, AutoProcessor, LlavaForConditionalGeneration
 
 from vllm import LLM, SamplingParams
+from vllm.config import VisionLanguageConfig
 from vllm.transformers_utils.tokenizer import get_tokenizer
 
 _TEST_DIR = os.path.dirname(__file__)
 _TEST_PROMPTS = [os.path.join(_TEST_DIR, "prompts", "example.txt")]
 _LONG_PROMPTS = [os.path.join(_TEST_DIR, "prompts", "summary.txt")]
 
+_PIXEL_VALUES_FILES = [
+    "images/stop_sign_pixel_values.pt", "images/cherry_blossom_pixel_values.pt"
```

Comment:
Can we generate these programmatically from the `.jpg` files and not check them in?

Reply:
Your comment makes me think I need to document what line of code output do pixel_values and image_features correspond to.
As to a programmatically way of generating these, while pixel_values is easy to do, image_features is not.

Reply:
Added more comments under `llava.py`.